### PR TITLE
fix 'Click on a link on google.com redirects to non-proxied page via playground in IE only' (close #838)

### LINKED ISSUE #838: Click on a link on google.com redirects to non-proxied page via playground in IE only
[screencast](http://screencast.com/t/6Z92CYsj)

### PR DESCRIPTION
**Pull request by miherlosev**
@churkin @LavrovArtem @VasilyStrelyaev

### CODE REVIEW

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

File: src/client/sandbox/node/element.js [hunk 1]
```diff
@@ -619,5 +619,10 @@ export default class ElementSandbox extends SandboxBase {
                 this._setValidBrowsingContextOnClick(el);
                 break;
         }
+
+        // NOTE: we need to reprocess on client side tags that doesn't processed on server
+        // see usage of the Parse5DomAdapter.needToProcessUrl
```
Comment (VasilyStrelyaev) [inline src/client/sandbox/node/element.js]:
```
        // NOTE: we need to reprocess a tag client-side if it wasn't processed on the server.
        // See the usage of Parse5DomAdapter.needToProcessUrl
```

**Comment by miherlosev** [comment]
@testcafe-build-bot \retest

**Comment by miherlosev** [comment]
FPR

**Pull request merged**